src/main/java/cz/cvut/kbss/study/model/PatientRecord.java

```diff
@@ -59,6 +59,9 @@ public class PatientRecord implements Serializable, HasOwlKey, HasUri {
     @OWLObjectProperty(iri = Vocabulary.s_p_has_phase, cascade = {CascadeType.MERGE}, fetch = FetchType.EAGER)
     private RecordPhase phase;
 
+   @OWLDataProperty(iri = Vocabulary.s_p_has_reject_message)
```

Comment:
Not sure why you call it a message, maybe because it is associated with frontend feature ?! ...

As this is stored in ontology (=triple store) I suggest to make it more understandable by:

`has-reject-message` --> `reject-reason`  (also applying naming convention, see below)

Reply:
here is also my comment from which it was not very clear :( -- https://github.com/kbss-cvut/record-manager-ui/issues/205#issuecomment-2431753652

Reply:
done, I renamed rejectMessage -> rejectReason